src/abstract.js

```diff
@@ -242,12 +242,12 @@ Sk.abstr.numOpAndPromote = function (a, b, opfn) {
     }
 
     if (a.constructor === Sk.builtin.lng) {
-//      if (b.constructor == Sk.builtin.nmber)
-//          if (b.skType == Sk.builtin.nmber.float$) {
-//              var tmp = new Sk.builtin.nmber(a.tp$str(), Sk.builtin.nmber.float$);
-//              return [tmp, b];
-//          } else
-//              return [a, b.v];
+        //if (b.constructor == Sk.builtin.nmber)
```

Comment:
I would prefer that you revert these changes.  It is so confusing when the only differences are with whitespace.   It makes it difficult to understand what is true implementation of the PR and what your editor has messed up.